roles/tests/test_utils.py

```diff
@@ -0,0 +1,103 @@
+"""
+Test the roles utils
+"""
+
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from django.core.management import call_command
+from django.utils.text import slugify
+from guardian.shortcuts import get_perms
+from mock import patch
+
+from learningresources.api import create_repo
+from learningresources.tests.base import LoreTestCase
+from roles.permissions import RepoPermission, GroupTypes
+from roles import utils
+
+
+class TestRoleUtils(LoreTestCase):
+    """
+    Tests for the roles Utils
+    """
+    # pylint: disable=too-many-instance-attributes
```

Comment:
here it's not necessary: it was a leftover of code I copied, but in roles/tests/test_api.py is necessary:
pylint is complain about too many `self` assignment in the `__init__` function